site/app/controllers/admin/UsersController.php

```diff
@@ -411,7 +411,20 @@ public function updateRegistrationSections() {
             if (User::validateUserData('registration_section', $_POST['delete_reg_section'])) {
                 // DELETE trigger function in master DB will catch integrity violation exceptions (such as FK violations when users/graders are still enrolled in section).
                 // $num_del_sections indicates how many DELETEs were performed.  0 DELETEs means either the section didn't exist or there are users still enrolled.
-                $num_del_sections = $this->core->getQueries()->deleteRegistrationSection($_POST['delete_reg_section']);
+                //why doesn't $this->core->getConfig()->getCoursePath(); work?
+                $fp = '/var/local/submitty/courses/f19/sample/uploads/course_materials_file_data.json';
```

Comment:
You should `$this->core->getConfig()->getCoursePath() . '/uploads/course_materials_file_data.json';` 
since the course path is something that could be configured.